npm/react/plugins/babel/index.d.ts

```diff
@@ -10,6 +10,11 @@ declare namespace legacyDevServer {
     setWebpackConfig?(config:Configuration): Configuration
   }
 
+  /**
+   * Type helper to make writing `CypressBabelDevServerConfig` easier
+   */
+  export function defineDevServerConfig(babelDevServerConfig: CypressBabelDevServerConfig): CypressBabelDevServerConfig
```

Comment:
For consistency, the first argument to `defineDevServerConfig` should be called `devServerConfig` in all modules.